.appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 3.0.0.99.{build}
+version: 3.1.0.{build}
```

Comment:
/gha run r-valgrind

Reply:
valgrind is OK.
https://github.com/microsoft/LightGBM/runs/1402192963?check_suite_focus=true

@jameslamb 
Seems we should update the number of bytes in out script

> 336 bytes in 1 blocks are possibly lost

https://github.com/microsoft/LightGBM/blob/742d72f8bb051105484fd5cca11620493ffb0b2b/.ci/test_r_package_valgrind.sh#L63

Reply:
👍 ok I can do that. It doesn't have to be done before this release though.